src/ralph/admin/tests/tests_functional.py

```diff
@@ -216,10 +216,10 @@ class ChangeListTest(TestCase):
 
     def setUp(self):
         super().setUp()
-        manufacturer = Manufacturer.objects.create(
+        manufacturer = TestManufacturer.objects.create(
```

Comment:
There used to be 2 Manufacturer models. The model used in tests has been renamed to TestManufacturer